admin_manual/installation/nginx_configuration.rst

```diff
@@ -80,8 +80,10 @@ Nginx Configuration
       rewrite ^/.well-known/host-meta /public.php?service=host-meta last;
       rewrite ^/.well-known/host-meta.json /public.php?service=host-meta-json last;
```

Comment:
Mhhh, a local Apache Installation is also doing the same. Thats one big mess :-/

```
curl -i http://127.0.0.1/.well-known/host-meta.json

HTTP/1.0 404 Not Found
Date: Tue, 13 Oct 2015 14:22:32 GMT
Server: Apache
X-Powered-By: PHP/5.5.9-1ubuntu4.13
Set-Cookie: ocsz3lwy561l=4mo8jbiki1lrbgnqcip3504d37; path=/; HttpOnly
Expires: Thu, 19 Nov 1981 08:52:00 GMT
Cache-Control: no-store, no-cache, must-revalidate, post-check=0, pre-check=0
Pragma: no-cache
Content-Security-Policy: default-src 'self'; script-src 'self' 'unsafe-eval'; style-src 'self' 'unsafe-inline'; frame-src *; img-src *; font-src 'self' data:; media-src *; connect-src *
X-XSS-Protection: 1; mode=block
X-Content-Type-Options: nosniff
X-Frame-Options: Sameorigin
X-Robots-Tag: none
Content-Length: 0
Connection: close
Content-Type: text/html; charset=UTF-8
```